2023-10-01 13:05:37.840290+02:00 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/goit/goitJava/IdeaProjects/JavaDeveloper/JavaDHomework4/test.mv.db [2.2.224/7]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at ua.homework.database.Database.<init>(Database.java:16)
	at ua.homework.database.Database.<clinit>(Database.java:10)
	at ua.homework.request.DatabasePopulateService.main(DatabasePopulateService.java:19)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/goit/goitJava/IdeaProjects/JavaDeveloper/JavaDHomework4/test.mv.db [2.2.224/7]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 20 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/goit/goitJava/IdeaProjects/JavaDeveloper/JavaDHomework4/test.mv.db [2.2.224/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.lockFileChannel(SingleFileStore.java:143)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:117)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 14 more
2023-10-01 13:42:06.832924+02:00 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/goit/goitJava/IdeaProjects/JavaDeveloper/JavaDHomework4/test.mv.db [2.2.224/7]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at ua.homework.database.Database.<init>(Database.java:16)
	at ua.homework.database.Database.<clinit>(Database.java:10)
	at ua.homework.request.DatabaseQueryService.readFileLongrestProject(DatabaseQueryService.java:83)
	at ua.homework.request.DatabaseQueryService.findLongrestProject(DatabaseQueryService.java:115)
	at ua.homework.request.DatabaseQueryService.main(DatabaseQueryService.java:118)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/goit/goitJava/IdeaProjects/JavaDeveloper/JavaDHomework4/test.mv.db [2.2.224/7]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/goit/goitJava/IdeaProjects/JavaDeveloper/JavaDHomework4/test.mv.db [2.2.224/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.lockFileChannel(SingleFileStore.java:143)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:117)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 16 more
2023-10-01 13:47:24.478573+02:00 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/goit/goitJava/IdeaProjects/JavaDeveloper/JavaDHomework4/test.mv.db [2.2.224/7]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at ua.homework.database.Database.<init>(Database.java:16)
	at ua.homework.database.Database.<clinit>(Database.java:10)
	at ua.homework.request.DatabaseQueryService.readFileLongrestProject(DatabaseQueryService.java:83)
	at ua.homework.request.DatabaseQueryService.findLongrestProject(DatabaseQueryService.java:115)
	at ua.homework.request.DatabaseQueryService.main(DatabaseQueryService.java:118)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/goit/goitJava/IdeaProjects/JavaDeveloper/JavaDHomework4/test.mv.db [2.2.224/7]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/goit/goitJava/IdeaProjects/JavaDeveloper/JavaDHomework4/test.mv.db [2.2.224/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.lockFileChannel(SingleFileStore.java:143)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:117)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 16 more
2023-10-01 13:48:36.825686+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "clientName" not found [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3518)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at ua.homework.request.DatabaseQueryService.readFileLongrestProject(DatabaseQueryService.java:92)
	at ua.homework.request.DatabaseQueryService.findLongrestProject(DatabaseQueryService.java:115)
	at ua.homework.request.DatabaseQueryService.main(DatabaseQueryService.java:118)
2023-10-01 13:49:00.655661+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "clientName" not found [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3518)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at ua.homework.request.DatabaseQueryService.readFileLongrestProject(DatabaseQueryService.java:92)
	at ua.homework.request.DatabaseQueryService.findLongrestProject(DatabaseQueryService.java:115)
	at ua.homework.request.DatabaseQueryService.main(DatabaseQueryService.java:118)
2023-10-01 13:49:54.059476+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "clientName" not found [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3518)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at ua.homework.request.DatabaseQueryService.readFileLongrestProject(DatabaseQueryService.java:92)
	at ua.homework.request.DatabaseQueryService.findLongrestProject(DatabaseQueryService.java:115)
	at ua.homework.request.DatabaseQueryService.main(DatabaseQueryService.java:118)
2023-10-01 13:51:28.847437+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "clientName" not found [42122-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:514)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3518)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:301)
	at ua.homework.request.DatabaseQueryService.readFileLongrestProject(DatabaseQueryService.java:92)
	at ua.homework.request.DatabaseQueryService.findLongrestProject(DatabaseQueryService.java:115)
	at ua.homework.request.DatabaseQueryService.main(DatabaseQueryService.java:118)
2023-10-01 14:57:25.284737+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 14:57:25.289741+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 14:57:25.300124+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 14:57:25.311124+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 14:58:43.307792+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 14:58:43.310774+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 14:58:43.343781+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 14:58:43.380854+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:02:00.344988+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:02:00.346990+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:02:00.359283+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:02:00.368398+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:07:36.212648+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:07:36.215624+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:07:36.224674+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:07:36.234639+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:08:34.988766+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:08:34.988766+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:08:35.004418+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:08:35.004418+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:28:05.600881+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:28:05.609670+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:28:05.621326+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:28:05.623356+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:30:25.698089+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:30:25.718090+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:30:25.743093+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:30:25.748095+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:31:25.816092+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:31:25.819096+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:31:25.827115+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:31:25.838296+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:36:33.930654+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:36:33.932675+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:36:33.943655+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:36:33.959655+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:38:46.586849+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:38:46.609854+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:38:46.630896+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2023-10-01 15:38:46.633852+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
